_data/changelogs/component-icon.yml

```diff
@@ -2,6 +2,12 @@ title: Icon
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
```

Comment:
**chore**: chore: Add tomorrow's date.

```suggestion
  - date: 2024-12-18
```

Reply:
Updated in [a386105](https://github.com/uswds/uswds-site/pull/2991/commits/a386105edfb98a0e8f218fd6b8e4eada5dfb1e55)